### PR TITLE
Modal header

### PR DESCRIPTION
**Pull request by xtina-starr**
Renames LoginHeader to ModalHeader as mentioned in the [title component PR](https://github.com/artsy/reaction-force/pull/32#discussion_r103729954). This also addresses' @alloy's [comment](https://github.com/artsy/reaction-force/pull/32#discussion_r103759753) about not needing to create the title component inside of the header component. Finally, I didn't for see any case where the modal header would need to have a state so I refactor it into a stateless component.

### CODE REVIEW

File: src/__stories__/config.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const { configure } = require("@kadira/storybook")
 function loadStories() {
   require("../components/__stories__/artwork")
   require("../components/__stories__/artwork_grid")
-  require("../components/__stories__/login_header")
+  require("../components/__stories__/modal_header")
```
Comment (yuki24) [inline src/__stories__/config.js]: This is unrelated, but I wonder if there is any way to automatically load all the stories in `../components/__stories__/`.

Comment (alloy) [inline src/__stories__/config.js]: I had looked into this, the issue is that this file gets evaluated in both Node and the browser and thus globbing a FS at that point isn’t that straightforward.

However, FuseBox does have an in-browser API that we can use to glob, so maybe we should conditionally use either the `node-glob` package or the FuseBox API.

Comment (yuki24) [inline src/__stories__/config.js]: Understood. I'm ok with maintaining this manually for now.

**Comment by broskoski** [comment]
LGTM2

**Comment by yuki24** [comment]
LTGM3

**Comment by broskoski** [comment]
Gonna close and reopen to see if the build success triggers.

**Pull request merged**